### PR TITLE
Clarify documentation regarding installation media

### PR DESCRIPTION
**Pull request by Zugschlus**
This is the result of the discussion in
https://community.theforeman.org/t/need-use-case-for-different-installation-media-for-single-operating-system/8320

### CODE REVIEW

File: _includes/manuals/1.17/4.4.2_prov_media.md [hunk 1]
```diff
@@ -12,3 +12,10 @@ Be sure to set the Family for the Media
...
+
+### Use case
+
+Most sites are well-served with having a 1:1 relationship between operating system and installation media. This will obviously work. However, there is a bunch of use cases that make sense to have more than one set of installation media per operating system. The most common case of this is when a site has datacenters in multiple locations and would like to have the systems automatically select the mirror closed to their location. This can be enforced by only allowing the "correct" location for a given set of installation media.
+
+Another possible use case is to use differnet of installation media for staging of different operating system versions. This is, however significantly more bumpy since this raises a bunch of issues with the version field in the actual operating system and might need other adjustments in the OS deployment process since, for example, the files that are actually used for the tftp boot during network installation are connected to the operating system and not the installation media set. Be prepared to change many templates if you decide doing so.
```
Comment (ekohl) [inline _includes/manuals/1.17/4.4.2_prov_media.md]: s/differnet/different/

File: _includes/manuals/1.17/4.4.2_prov_media.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
-The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set.
+The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set. In the Red Hat World, installation media are a resource for the installation repo.
```
Comment (ekohl) [inline _includes/manuals/1.17/4.4.2_prov_media.md]: This addition doesn't clarify it much for me.

Comment (lzap) [reply _includes/manuals/1.17/4.4.2_prov_media.md]: Yeah I'd drop this.

**Review by dLobatog — CHANGES_REQUESTED** [review]
@Zugschlus Setting to waiting on contributor because it's missing a few things commented inline.

File: _includes/manuals/1.17/4.4.2_prov_media.md (hunk 2 above)
Comment (Zugschlus) [reply _includes/manuals/1.17/4.4.2_prov_media.md]: @lzap, the clarification was taken verbatim from your own suggestion in https://community.theforeman.org/t/need-use-case-for-different-installation-media-for-single-operating-system/8320, and it doesn't clarify much for me, but you said it was important to understand, so I put the text in my patch.

File: _includes/manuals/1.17/4.4.2_prov_media.md (hunk 1 above)
Comment (Zugschlus) [reply _includes/manuals/1.17/4.4.2_prov_media.md]: I'll file a new PR for the obvious one-letter change. Wouldn't it have been easier for all participants to simply merge and then do the one-letter commit?

**Comment by Zugschlus** [comment]
I think that the required changes are done now, I don't know whether I can do anything to get rid of the "waiting for changes" status of the PR.

File: _includes/manuals/1.17/4.4.2_prov_media.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 
-The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set.
+The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set. It is important to understand that, in the Red Hat World, installation media are a resource for the installation repo, not for the updates repo.
```
Comment (lzap) [inline _includes/manuals/1.17/4.4.2_prov_media.md]: How about: Make sure to use URL with OS installer images, some Linux distributions publish repositories containing no installers (e.g. fedora-updates).

Comment (Zugschlus) [reply _includes/manuals/1.17/4.4.2_prov_media.md]: much better for me.

Comment (Zugschlus) [reply _includes/manuals/1.17/4.4.2_prov_media.md]: >The Installation Media represents the web URL from where the installation packages
>can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when
>Foreman is first installed. However, it is best to edit the media you are going to
>use and ensure the Family is set. Make sure to use URLs that point to
>actual OS installer repositories. The installer-less repositories published by
>some Linux distributions (e.g. fedora-updates) won't work here

how about that?

Comment (lzap) [reply _includes/manuals/1.17/4.4.2_prov_media.md]: Better, thanks. Also "However, it is best to edit the media you are going to
use and ensure the Family is set." is a bit weird to me. Perhaps "Be sure to select closest or local mirror and ensure family is set correctly."

**Comment by GregSutcliffe** [comment]
+1 from me. @dLobatog  / @lzap is there anything else pending here?

**Review by lzap — APPROVED** [review]
No comments there, thanks.

**Comment by GregSutcliffe** [comment]
Thanks @Zugschlus et al!

**Pull request merged**